rollup/rollup.js

```diff
@@ -0,0 +1,7 @@
+const {link} = require("@rules_js/linker");
```

Comment:
can we use `--require` to make the linker call happen from the shell launcher, so we don't need to modify rollup?

Reply:
yes. I just need to change the linker a bit to call the linker function when it is being called by node directly rather than being imported.